kplclient.go

```diff
@@ -62,6 +77,57 @@ func processChannel() {
 	}
 }
 
+func (c *KPLClient) processErrMessage() {
+	l, err := net.Listen("tcp", c.ErrPort)
+	if err != nil {
+		fmt.Println("Error listening to error port:", err.Error())
+		return
+	}
+
+	// Close the listener when the application closes.
+	defer l.Close()
+
+	log.Println("Waiting for client connection")
+	conn, err := l.Accept()
+	if err != nil {
+		log.Println("Error accepting error connection: ", err.Error())
+		return
+	}
+
+	log.Println("Connection established")
+
+	for {
+		//Read from err to socket
+		content, err := Read(conn)
+		if err != nil {
+			log.Printf("Listener: Read error: %v", err)
+			time.Sleep(time.Millisecond)
```

Comment:
ditto?

Reply:
This one I added to not flood the system with log as we are doing this in infinite loop. Should I remove it?

Reply:
Hmm, well if `Read(conn)` returns an error should we really be continuing? Eventually the connection may need to be reestablished.

Reply:
Make sense. I will remove the sleep.